Production_scripts/convert_gp_lookup.R

```diff
@@ -0,0 +1,126 @@
+#####################################################
+# GP Lookup
+# Author: Catherine Holland
+# Date: February 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input - Open Data, GP Practice Details, Postcode Lookup
+# Description - Build the GPprac Lookup.
+#               Pulls from the Open Data Platform and lookups.
+#####################################################
+
+
+## packages ##
+library(dplyr)
+library(tidyr)
+library(haven)
+library(phsopendata)
+library(janitor)
+library(fs)
+
+
+## get data ##
+
+# Retrieve the latest resource from the dataset
+opendata <-
+  get_dataset("gp-practice-contact-details-and-list-sizes",
+    max_resources = 1
+  ) %>%
+  clean_names() %>%
+  # Filter and save
+  select(
+    gpprac = practice_code,
+    practice_name = gp_practice_name,
+    postcode,
+    cluster = gp_cluster
+  ) %>%
+  # Sort for SPSS matching
+  arrange(gpprac) %>%
+  # Write as an SPSS file
+  write_sav(
+    path(
+      # lookup_dir,
+      get_lookups_dir(),
+      paste0("practice_details_", latest_update()),
+      ext = "zsav"
+    ),
+    compress = TRUE
+  )
```

Comment:
This is saving to the wrong lookups path which is stored in cl-out. The path we want is \\stats\hscdiip\SLF_Extracts\Lookups which already has a function called get_practice_details_path(). I've amended this and pushed a commit
```suggestion
  write_sav(
    path(
      # lookup_dir,
      get_practice_details_path()
    ),
    compress = TRUE
  )
```